Mathlib/NumberTheory/NumberField/Units/Basic.lean

```diff
@@ -55,6 +55,26 @@ theorem NumberField.isUnit_iff_norm [NumberField K] {x : 𝓞 K} :
   convert (RingOfIntegers.isUnit_norm ℚ (F := K)).symm
   rw [← abs_one, abs_eq_abs, ← Rat.RingOfIntegers.isUnit_iff]
 
+theorem NumberField.IsOfFinOrder_iff_eq_one_or_neg_one_of_odd_finrank [NumberField K]
```

Comment:
I am a bit confused by this statement. I would rather prove something like: under `(h : Odd (Module.finrank ℚ K))`, if `x : torsion K`, then `x = 1` or `x = -1`.

Reply:
Also, this result should be in the `torsion` section.